### ISSUE TITLE
obsolete GO:0099110    microtubule polymerization based protein transport

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#20318]
Follow up from
https://github.com/geneontology/go-ontology/issues/20258

Please obsolete
GO:0099110    microtubule polymerization based protein transport to cell tip cortex
(this has only one annotation and I don't think this is polymerization based. I think tea1 is a normal cargo)
and
GO:0099112    microtubule polymerization based protein transport
the parent.

I wasn't sure about this at the time, and I have never seen it anywhere else.

This will simplify a little. Much more to do here.

**Comment by ValWood** [comment]
I would be inclined to get rid of "microtubule polymerization based movement" also 
and keep only "microtubule-based movement" and the grand kids.

**Comment by pgaudet** [comment]
I agree, I made the same comment in https://github.com/geneontology/go-ontology/issues/20258#issuecomment-724511362
